app/controllers/state_file/faq_controller.rb

```diff
@@ -2,10 +2,18 @@ class StateFile::FaqController < ApplicationController
   layout "state_file"
 
   def index
+    intake_start = Rails.configuration.state_file_start_of_open_intake
+    active_tax_year = if (app_time.year == intake_start.year) && app_time < intake_start
+                        # if the start of intake is set to this year and it is currently before that date, we should still show the previous year's states
+                        Rails.configuration.statefile_current_tax_year - 1
+                      else
+                        Rails.configuration.statefile_current_tax_year
+                      end
+    visible_state_code_names = StateFile::StateInformationService.state_code_to_name_map.filter { |code, name| StateFile::StateInformationService.filing_years(code).include?(active_tax_year) }
```

Comment:
since `name` is unused could you use `active_state_codes` directly instead of `state_code_to_name_map`? Minor change, but one fewer loop/data transformation every time the page is hit.

Reply:
name is used! the original controller action also uses `state_code_to_name_map` and in the view it displays the state name as part of the title of the FAQ section